armotypes/notificationreporttypes.go

```diff
@@ -0,0 +1,151 @@
+	RegistriesScanned                   []RegistryScanned   `json:"registriesScanned" bson:"registriesScanned"`
+}
+
+type ClusterScanned struct {
+	Cluster ClusterResourceScanned `json:"cluster" bson:"cluster"`
+}
+
+type RegistryScanned struct {
+	Registry ResourceScanned `json:"registry" bson:"registry"`
+}
+
+type RepositoryScanned struct {
+	Repository ResourceScanned `json:"repository" bson:"repository"`
+}
+
+type ClusterResourceScanned struct {
+	FullName  string                     `json:"fullName" bson:"fullName"`
+	ShortName string                     `json:"shortName" bson:"shortName"`
+	Resource  map[string]ResourceDetails `json:"resourcesToDetails" bson:"resourcesToDetails"`
+}
+
+type ResourceScanned struct {
+	Name     string                     `json:"name" bson:"name"`
+	Resource map[string]ResourceDetails `json:"resourcesToDetails" bson:"resourcesToDetails"`
+}
+
+type ResourceDetails struct {
+	FailedResourcesNumber int `json:"failedResourcesNumber" bson:"failedResourcesNumber"`
+}
+
+type Vulnerability struct {
+	Name               string                         `json:"name" bson:"name"`
+	ImgHash            string                         `json:"imageHash" bson:"imageHash"`
+	ImgTag             string                         `json:"imageTag" bson:"imageTag"`
+	RelatedPackageName string                         `json:"packageName" bson:"packageName"`
+	PackageVersion     string                         `json:"packageVersion" bson:"packageVersion"`
+	Link               string                         `json:"link" bson:"link"`
+	Description        string                         `json:"description" bson:"description"`
+	Severity           string                         `json:"severity" bson:"severity"`
+	SeverityScore      int                            `json:"severityScore" bson:"severityScore"`
+	Metadata           interface{}                    `json:"metadata" bson:"metadata"`
+	Fixes              VulFixes                       `json:"fixedIn" bson:"fixedIn"`
+	Relevancy          string                         `json:"relevant" bson:"relevant"`
+	UrgentCount        int                            `json:"urgent" bson:"urgent"`
+	NeglectedCount     int                            `json:"neglected" bson:"neglected"`
+	HealthStatus       string                         `json:"healthStatus" bson:"healthStatus"`
+	Categories         VulnerabilityCategory          `json:"categories" bson:"categories"`
+	ExceptionApplied   []VulnerabilityExceptionPolicy `json:"exceptionApplied,omitempty" bson:"exceptionApplied,omitempty"`
+}
+
+type VulFixes []FixedIn
+
+type VulnerabilityCategory struct {
+	IsRCE bool `json:"isRce" bson:"isRce"`
+}
+
+type FixedIn struct {
+	Name    string `json:"name" bson:"name"`
+	ImgTag  string `json:"imageTag" bson:"imageTag"`
+	Version string `json:"version" bson:"version"`
+}
+type TopVulItem struct {
+	Vulnerability   `json:",inline"`
+	WorkloadsCount  int64 `json:"workloadsCount" bson:"workloadsCount"`
```

Comment:
Vulnerability     is not used anywhere else besides in here - so why not adding these two fields to Vulnerability  and remove this struct?